src/GitVersionTask.Tests/packages.config

```diff
@@ -15,6 +16,21 @@
   <package id="NUnitTestAdapter" version="2.0.0" targetFramework="net45" />
   <package id="ObjectApproval" version="1.2.0" targetFramework="net45" />
   <package id="Shouldly" version="2.5.0" targetFramework="net45" />
+  <package id="System.Collections" version="4.0.0" targetFramework="net45" />
   <package id="System.Collections.Immutable" version="1.1.37" targetFramework="net45" />
+  <package id="System.Diagnostics.Debug" version="4.0.0" targetFramework="net45" />
+  <package id="System.Globalization" version="4.0.0" targetFramework="net45" />
+  <package id="System.IO" version="4.0.0" targetFramework="net45" />
```

Comment:
Ok, so NuGet v2 doesn't support the concept of transitive dependencies. So all dependencies get installed. With PCL/dnx the framework packages are listed as dependencies. 

NuGet v3 will fix this apparently.